frame/cere-staking/src/lib.rs

```diff
@@ -0,0 +1,599 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use codec::{Decode, Encode, HasCompact};
+use frame_support::{
+	parameter_types,
+	traits::{Currency, DefensiveSaturating, LockIdentifier, WithdrawReasons},
+	BoundedVec,
+};
+use scale_info::TypeInfo;
+use sp_runtime::{
+	traits::{AtLeast32BitUnsigned, CheckedSub, Saturating, Zero},
+	RuntimeDebug,
+};
+use sp_staking::EraIndex;
+use sp_std::prelude::*;
+
+pub use pallet::*;
+
+const CERE_STAKING_ID: LockIdentifier = *b"cnmstake"; // Cere Network Maintainer's stake
+
+/// The balance type of this pallet.
+pub type BalanceOf<T> =
+	<<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
+
+parameter_types! {
+	pub MaxUnlockingChunks: u32 = 32;
```

Comment:
Why `32`?

Reply:
It is the [same in the FRAME staking pallet](https://github.com/Cerebellum-Network/pos-network-node/blob/c026e5a11f3ced2fbaaca194c44e017b5e7e915a/frame/staking/src/lib.rs#L352) and will not introduce an unexpected behavior for the users familiar with regular Cere Network staking.